Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -209,18 +207,26 @@ public boolean registerAuthy(UUID uuid, String email, String phone, String count
         }
 
         // Redis
-        Redis.update(result, redisPool, redisConfigNode);
+        Redis.update(result, ConfigUtil.getRedisPool(), ConfigUtil.getRedisConfigNode());
 
         // RabbitMQ
-        RabbitMQ.broadcast(result, rabbitConnection);
+        try {
+            RabbitMQ.broadcast(result, getRabbitConnection());
```

Comment:
Not using the broadcast method, here?
Redis & Rabbit broadcast/update should probably be in one method, if we're going to make it a method anyway.